app/src/main/java/nl/tudelft/b_b_w/model/TrustValues.java

```diff
@@ -7,7 +7,7 @@
     /**
      * Predefined trust values
      */
-    INITIALIZED(20), REVOKED(0), VERIFIED(50), SUCCESFUL_TRANSACTION(10), FAILED_TRANSACTION(-10);
+    INITIALIZED(10), REVOKED(0), VERIFIED(20), SUCCESFUL_TRANSACTION(10), FAILED_TRANSACTION(-10);
```

Comment:
The initialized value is a value which has to be a bit above 0, but not too much and since the upper limit is 100, it will reach it far too fast. The values of succesful transaction ,failed transaction, verified iban have to be removed later